paimon-spark/paimon-spark-common/src/main/scala/org/apache/paimon/spark/PaimonAnalysis.scala

```diff
@@ -18,8 +18,10 @@
 package org.apache.paimon.spark
 
 import org.apache.spark.sql.SparkSession
-import org.apache.spark.sql.catalyst.plans.logical.LogicalPlan
+import org.apache.spark.sql.catalyst.plans.logical.{LogicalPlan, OverwritePartitionsDynamic}
 import org.apache.spark.sql.catalyst.rules.Rule
+import org.apache.spark.sql.execution.datasources.v2.DataSourceV2Relation
+import org.apache.spark.sql.paimon.commands.PaimonDynamicPartitionOverwriteCommand
```

Comment:
We have to put the class under `org.apache.spark.sql` so that some private methods can be used, and the current import clause is the result of automatic formatting